src/sidebar/components/share-dialog.js

```diff
@@ -8,15 +8,15 @@ function ShareDialogController($scope, $element, analytics, store) {
 
   function updateViaLink(frames) {
     if (!frames.length) {
-      self.viaPageLink = '';
+      self.sharePageLink = '';
       return;
     }
 
     // Check to see if we are on a via page. If so, we just return the URI.
     if (frames[0].uri.indexOf(VIA_PREFIX) === 0) {
```

Comment:
@dwhly This is where we were using the VIA_PREFIX var. We are checking to see if the user is already on a via page in which case we don't need to create a bouncer link for it.
The environment variable to determine the bouncer prefix i.e `hyp.is` is relevant to the `else` block of this code which will create a bouncer link instead of a via link for the url in question.

Reply:
Yeah, mostly just talking about other places we might still be using VIA_PREFIX... not this one.  I know you're changing it here to use bouncer. Sorry to interrupt. :)